.gitattributes

```diff
@@ -1,2 +0,0 @@
-**/zz_generated*.go linguist-generated=true
-pkg/clientset/** linguist-generated=true
```

Comment:
We should keep this file so that the automatically generated file will not be shown in the diff of GitHub pr

Reply:
this is the pages branch so it's a bit a different but sure I can keep it.